src/components/util/Counter.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import Button from '../common/Button';
+
+function Counter({ count, onClick }) {
+  return (
+    <Button onClick={() => onClick()}>
```

Comment:
```suggestion
    <Button onClick={onClick}>
```

이 경우엔 이렇게만 처리해줘도 좋을 것 같아요

Reply:
말씀해주신 코드
```
<Button onClick={onClick}>
```
onClick에 onClick함수를 바로 prop으로 넘기면 Click me가 작동이 되지않습니다. 

() => onClick은 어 되네? onClick은 어 안되네? 라고 넘기기에는... 
놓치고 있는 큰 개념이 어딘가...
bind가 왜 안되는 것이지...ㅎㅎ

원인을 찾아봐서 정리 및 진단 해볼께요!

Reply:
제가 Counter 함수만 보고 리뷰를 드렸던 것 같네요 ㅎㅎ;

혹시 당시에 작동이 되지 않을때 에러나 발생했던 현상이 어떤게 있을까요?

일단 제가 저 당시 커밋 내용 기반으로 테스트를 한번 해봤는데, NAN이 찍히더라구요.
그 이유는 Button 컴포넌트에서 children을 넘겨받은 onClick에 더해주고 있어서 숫자와 문자열이 더해지니 당연히 NAN이 찍히는 것 같아요. 
혹시 다른 부분이 있다면 말씀해주세요~ :)